projects/storefrontlib/src/cms-components/asm/loader/asm-loader.component.html

```diff
@@ -0,0 +1,8 @@
+<!-- 
```

Comment:
E2E tests "storefront should have ASM feature enabled" needs to be updated.  Probably only needs to change the name of the main tag it should expect when asm is enabled on the storefront.